aerogear-android-push/src/main/java/org/jboss/aerogear/android/unifiedpush/gcm/UnifiedPushConfig.java

```diff
@@ -245,7 +245,7 @@ public UnifiedPushConfig setAlias(String alias) {
      * 
      */
     public UnifiedPushConfig setCategories(List<String> categories) {
-        this.categories = new ArrayList<String>(categories);
+        this.categories = categories;
```

Comment:
Only because it does not work. I asked me the same when community send a bug report. I did some tests and it does not work and I have no idea why :)